timber-sample/src/main/java/com/example/timber/ExampleApp.java

```diff
@@ -15,6 +15,10 @@
     } else {
       Timber.plant(new CrashReportingTree());
     }
+
+    int size = 10;
+    int i = 5;
+    Timber.i("Position %d", size - i);
```

Comment:
Were you able to get this to warn before your change? I wasn't able to get the lint checks to run on the example app.